migrations/sqls/20190726234757-programs-up.sql

```diff
@@ -0,0 +1,155 @@
+/* Replace with your SQL commands */
+CREATE TABLE programs (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  programName varchar(255),
+  programDescription varchar(1028)
+);
+
+INSERT INTO programs (programName, programDescription) VALUES
+  ("Preventive Health", "Preventive Health"),
+  ("Nutrition / CRYS / SNAP", "Nutrition / CRYS / SNAP"),
+  ("Immigration", "Immigration"),
+  ("Financial Education / Coaching", "Financial Advising / Coaching"),
+  ("Workers' Rights", "Workers' Rights"),
+  ("Youth Groups", "Youth Groups"),
+  ("Family Support", "Family Support Services"),
+  ("Community Engagement and Organizing", "Community Engagement and Organizing")
+;
+
+ALTER TABLE services
+  ADD COLUMN programId INT,
+  ADD FOREIGN KEY programIdFk(programId) REFERENCES programs(id);
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "Citizenship";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "Family Petition";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Workers' Rights"
+) WHERE serviceName = "Workers' Rights and Safety";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "DACA";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Youth Groups"
+) WHERE serviceName = "Youth Groups";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Nutrition / CRYS / SNAP"
+) WHERE serviceName = "SNAP";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Preventive Health"
+) WHERE serviceName = "Chronic Disease Testing";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Nutrition / CRYS / SNAP"
+) WHERE serviceName = "Nutrition";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Nutrition / CRYS / SNAP"
+) WHERE serviceName = "Grocery Store Tour";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Nutrition / CRYS / SNAP"
+) WHERE serviceName = "Cooking Classes";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Financial Education / Coaching"
+) WHERE serviceName = "Financial Coach";
```

Comment:
The June 2019 copy of the form seems to also have eliminated the separation between "Financial Coach" and "Financial Education".

Reply:
That google doc seems to sometimes have it split and sometimes not:

<img width="340" alt="image" src="https://user-images.githubusercontent.com/5524384/62478083-78571c00-b767-11e9-9a1e-1f54cf8ca4fe.png">

<img width="557" alt="image" src="https://user-images.githubusercontent.com/5524384/62478062-6e351d80-b767-11e9-82ba-634dce3eec8f.png">

<img width="734" alt="image" src="https://user-images.githubusercontent.com/5524384/62478009-578ec680-b767-11e9-8908-1e3ed4d89718.png">

Vicky's version of the programs and services *does* have them split. See https://docs.google.com/document/d/1fF-8KjJG16Yk7I_qG2X4FpydZRgFtM2yiyHmiLNjSjI/edit